imports/ui/signup/signup.jsx

```diff
@@ -9,14 +10,43 @@ import PasswordInput from '../components/password-input'
 import actions from './signup.actions'
 import LoginLayout from '../layouts/login-layout'
 import { emailValidator } from '../../util/validators'
+import CircularProgress from 'material-ui/CircularProgress'
+
+type Props = {
+  dispatch: (info: {}) => void,
+  userCreationState: {|
```

Comment:
Doesn't really have to fix anything here, but there's no added value for restricting the object to just these attributes with the `{|...|}` notation here